apollo-compiler/src/test/graphql/com/example/antlr_tokens/type/CustomScalars.kt

```diff
@@ -0,0 +1,16 @@
+// AUTO-GENERATED FILE. DO NOT MODIFY.
+//
+// This class was automatically generated by Apollo GraphQL plugin from the GraphQL queries it found.
+// It should not be modified by hand.
+//
+package com.example.antlr_tokens.type
+
+import com.apollographql.apollo.api.CustomScalar
+
+object CustomScalars {
```

Comment:
Didn't we have PR into 2.x branch that introduces some interface for `CustomScalars` when it's generated as sealed class? Also I believe we had compiler option to generate this as sealed class, are we sure to change this behaviour in 3.x?

Reply:
This was about Enums and the `EnumValue` interface: https://github.com/apollographql/apollo-android/blob/d59f2a1b07c62b7094a8ecfd56dd57fe5f89cd63/apollo-api/src/commonMain/kotlin/com/apollographql/apollo/api/EnumValue.kt#L6

This should be backported to 3.0. as part of https://github.com/apollographql/apollo-android/pull/2813